### PR TITLE
MAINT: Fixed spelling and grammar in README.md

### PR DESCRIPTION
**Pull request by SYK-08**
### Checks

- [x] My commits follow the [commit guidelines](https://github.com/mumble-voip/mumble/blob/master/COMMIT_GUIDELINES.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Mumble is an Open Source, low-latency and high-quality voice-chat program
 written on top of Qt and Opus.
 
 There are two modules in Mumble; the client (mumble) and the server (murmur).
-The client works on Windows, Linux, FreeBSD, OpenBSD and macOS,
+The client works on Windows, Linux, FreeBSD, OpenBSD, and macOS,
```
Comment (Krzmbrzl) [inline README.md:18]: I don't think commas belong in front of an "and"

File: README.md [hunk 2]
```diff
@@ -43,16 +43,16 @@ We are using Weblate as a translation platform. [Register on Weblate](https://ho
 ### Writing plugins
 
 Mumble supports general-purpose plugins that can provide functionality that is not implemented in the main Mumble application. You can find more
-information on how this works and on how these have to be created in the [plugin documentation](docs/dev/plugins/README.md).
+information on how this works and how these have to be created in the [plugin documentation](docs/dev/plugins/README.md).
```
Comment (Krzmbrzl) [inline README.md:46]: I think the "on" in the original version is important :thinking:

**Comment by Krzmbrzl** [comment]
Furthermore, please make your commit be in line with our commit guidelines as linked in the checkbox you checked in your PR description. This would be a `MAINT` commit as it only touches files that are not associated with the Mumble software itself.

File: README.md (hunk 2 above)
Comment (ppfeufer) [reply README.md:46]: The "on" is redundant and can be removed.
image

Or to put it syntactically:
```
… information on [how this works and how these have to be created] in the …
```
equals
```
… information on [this matter] in the …
```

File: README.md (hunk 1 above)
Comment (SYK-08) [reply README.md:18]: We should add a comma before 'and' when it connects two independent clauses.

Comment (ppfeufer) [reply README.md:18]: Both versions are correct here, although the expression without a comma before the `and` is more common. 
The "commas don't belong in front of `and`" rule is more of a German thing and not even true there all the time.

File: README.md (hunk 2 above)
Comment (SYK-08) [reply README.md:46]: > I think the "on" in the original version is important :thinking: 

Not really, it is just taking extra space there

**Comment by SYK-08** [comment]
> Furthermore, please make your commit be in line with our commit guidelines as linked in the checkbox you checked in your PR description. This would be a `MAINT` commit as it only touches files that are not associated with the Mumble software itself.

Will do next time for sure 😁

File: README.md (hunk 1 above)
Comment (jvbsl) [reply README.md:18]: It's called an oxford comma(or some other names) and is indeed a valid way for lists(I personally do prefer it)

Comment (Krzmbrzl) [reply README.md:18]: Interesting. Never knew about that one ^^

**Comment by Krzmbrzl** [comment]
We need the changed commit message for this PR already as we don't want to add any non-conforming commits to the history (this breaks the script that we use to auto-summarize changes).

**Review by Krzmbrzl — APPROVED** [review]
I took the liberty and changed the commit message for you

**Comment by davidebeatrici** [comment]
The verb tense should be present, though.

**Comment by Krzmbrzl** [comment]
> The verb tense should be present, though.

Huh? I don't think I understand what you mean :eyes:

**Comment by davidebeatrici** [comment]
`MAINT: Fix spelling and grammar in README.md`

**Comment by Krzmbrzl** [comment]
You mean Fixed -> Fix? We always mix the tense of this kind of stuff... I don't think there is any consistency in how we use that 👀

**Comment by Krzmbrzl** [comment]
Thank you for your contribution :+1:

**Pull request merged**